### DISCUSSION TITLE
[small fix] [UI Accuracy] [clock] reorder title and add top bar to clock flyout

### DISCUSSION THREAD

**Discussion by CorruptionVR** [DustinBrett/daedalOS#385] [Ideas]
- [ ] In windows, the title for the clock shows as "Weekday-Long, Month-Long Day-Short, Year-Long". Ex. "Thursday, November 9, 2023", but in daedalOS, it shows as "Month-Long Day-Short, Year-Long, Weekday-Long". Ex. "November 9, 2023, Thursday"
- [ ] windows has a top bar on the clock flyout that shows the hour, minute, seconds in large numbers, and AM/PM in small letters, and date in long form which resets focus on the calendar to the current date when clicked
Windows
![image](https://github.com/DustinBrett/daedalOS/assets/78228404/c062f27f-fa45-4108-9490-2682ddad65f7)
daedalOS
![image](https://github.com/DustinBrett/daedalOS/assets/78228404/f9f89b86-d553-4aba-9f92-fe65cd3272a2)

**Comment by DustinBrett** [comment]
For the first point, if you mean on hover, it actually matches Windows 10. Perhaps in other regions it is different.

Windows
![image](https://github.com/DustinBrett/daedalOS/assets/16656318/258f6c3d-24e0-4e6a-a56d-fe5290e67e37)

daedalOS
![image](https://github.com/DustinBrett/daedalOS/assets/16656318/d9b10331-b405-45ed-8dfc-bd3be1737709)

For the 2nd point, I did indeed decide to remove the top banner thing as I show seconds on the clock already and on hover you get the weekday. I thought the calendar was more simple like this so atm I kept it as a kind of minimal version.

Thanks for the suggestions and I will consider future tweaks/customizations.

**Reply by CorruptionVR** [reply]
oh, ok. I'm in the us, so it shows as weekday first for me.

**Reply by DustinBrett** [reply]
I think I have mine set to US also but I am in Canada. It's interesting that for you it shows the other way. I will look into it and can perhaps adjust this. I do have plans to add i18n to allow translation and localization of many parts of the app.
